stacker/config/__init__.py

```diff
@@ -35,5 +89,210 @@ def parse_config(raw_config, environment=None):
         buff.write(t.safe_substitute(environment))
 
     buff.seek(0)
-    config = yaml.load(buff)
+    return buff.read()
+
+
+def parse(raw_config):
+    """Parse a raw yaml formatted stacker config.
+
+    Args:
+        raw_config (str): the raw stacker configuration string in yaml format.
+
+    Returns:
+        :class:`Config`: the parsed stacker config.
+
+    """
+
+    # We have to enable non-strict mode, because people may be including top
+    # level keys for re-use with stacks (e.g. including something like
+    # `common_variables: &common_variables`).
+    #
+    # The unfortunate side effect of this is that it propagates down to every
+    # schematics model, and there doesn't seem to be a good way to only disable
+    # strict mode on a single model.
+    #
+    # If we enabled strict mode, it would break backwards compatibility, so we
+    # should consider enabling this in the future.
+    strict = False
+
+    return Config(yaml.safe_load(raw_config), strict=strict)
+
+
+def load(config):
+    """Loads a stacker configuration by modifying sys paths, loading lookups,
+    etc.
+
+    Args:
+        config (:class:`Config`): the stacker config to load.
+
+    Returns:
+        :class:`Config`: the stacker config provided above.
+
+    """
+
+    if config.sys_path:
+        logger.debug("Appending %s to sys.path.", config.sys_path)
+        sys.path.append(config.sys_path)
+        logger.debug("sys.path is now %s", sys.path)
+    if config.lookups:
+        for key, handler in config.lookups.iteritems():
+            register_lookup_handler(key, handler)
+    sources = config.package_sources
+    if sources is not None:
+        processor = SourceProcessor(
+            stacker_cache_dir=config.stacker_cache_dir
+        )
+        processor.get_package_sources(sources=sources)
+
     return config
+
+
+def dump(config):
+    """Dumps a stacker Config object as yaml.
+
+    Args:
+        config (:class:`Config`): the stacker Config object.
+        stream (stream): an optional stream object to write to.
+
+    Returns:
+        str: the yaml formatted stacker Config.
+
+    """
+
+    return yaml.safe_dump(
+        config.to_primitive(),
+        default_flow_style=False,
+        encoding='utf-8',
+        allow_unicode=True)
+
+
+def more_than_one(value):
```

Comment:
Derp. Fixed in https://github.com/remind101/stacker/pull/443/commits/fe9c8cf771a8f887148572090f4dd6c316005124